docs/jwst/associations/level3_asn_technical.rst

```diff
@@ -59,47 +59,55 @@ The following example will be used to explain the contents of an association::
         ]
     }
 
-.. _asn-association-meta-keywords:
+.. _asn-level3-meta-keywords:
 
 Association Meta Keywords
 -------------------------
 
-The following are the top-level, or meta, keywords of an association.
-
-program *optional*
-  Program number for which this association was created.
-
-target *optional*
-  Target ID for which this association refers to. DMS currently uses
-  the TARGETID header keyword in the stage 2 exposure files, but there
-  is no formal restrictions on value.
+The following are the informational, meta keywords of an association.
 
-asn_type *optional*
-  The type of association represented. See :ref:`asn-jwst-association-types`
-
-asn_id *optional*
+asn_id *required*
   The association id. The id is what appears in the :ref:`asn-jwst-naming`
 
-asn_pool *optional*
+asn_pool *required*
   Association pool from which this association was created.
 
 asn_rule *optional*
   Name of the association rule which created this association.
 
-degraded_status *optional*
-  Error status from the observation logs. If none the phrase "No
-  known degraded exposures in association." is used.
+asn_type *optional*
+  The type of association represented. See :ref:`asn-jwst-association-types`
 
-version_id *optional*
-  Version identifier. DMS uses a time stamp with the format
-  ``yyyymmddthhmmss``
-  Can be None or NULL
+code_version *optional*
+  The version of the generator which created this association. Typically this is the version
+  of the jwst python package.
 
 constraints *optional*
   List of constraints used by the association generator to create this
   association. Format and contents are determined by the defining
   rule.
 
+degraded_status *optional*
+  If any of the included members have an actual issue,
+  as reported by the ``exposerr`` keyword, ``degraded_status`` will have the
+  value ``One or more members have an error associated with them.`` If no errors
+  are indicated, the value will be ``No known degraded exposures in
+  association.``
+
+program *optional*
+  Program number for which this association was created.
+
+target *optional*
+  Target ID for which this association refers to. JWST currently uses
```

Comment:
Change grammar to "Target ID to which this association refers."

Reply:
✅